.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+script: travis_retry npm test
```

Comment:
It's not common when doing your tests, but it can be common when installing dependencies so it's usually a good idea to add that. I should do that actually, i completely forgot.